### PR TITLE
Update README and logo

### PR DESCRIPTION
**Pull request by robinellgren**
The README is currently not very "selling". It is not immediately clear why anyone should use this library. This PR adds a small logotype and makes the instructions more clear and selling (imo).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +121,10 @@ export const store = configureStore({
     getDefaultMiddleware().concat(api.middleware),
 })
 
+**6. Enjoy type-safe hooks.**
```
Comment (otahontas) [inline README.md:124]: I guess this was supposed to be another subheader?

Comment (otahontas) [reply README.md:124]: // 4. Use your typed hooks can be probably removed?

File: README.md [hunk 2]
```diff
@@ -1,42 +1,78 @@
...
 ### Installation
 
-Install the library and peer dependencies:
+**1. Install `trpc-rtk-query` and peer dependencies.**
 
-```sh
-npm install trpc-rtk-query @reduxjs/toolkit @trpc/client @trpc/server
+```bash
+# npm
+npm install trpc-rtk-query @reduxjs/toolkit @trpc/client
```
Comment (otahontas) [inline README.md:43]: `@trpc/server` is also peer dep

**Review by otahontas — CHANGES_REQUESTED** [review]
Very nice one, thank you! Just a couple of minor comments

File: README.md (hunk 1 above)
Comment (robinellgren) [reply README.md:125]: fixed!

**Review by otahontas — APPROVED** [review]

**Pull request merged**